bakerydemo/templates/tags/breadcrumbs.html

```diff
@@ -3,24 +3,25 @@
 
 <nav class="breadcrumb-container" aria-label="breadcrumb" role="navigation">
   <div class="container">
-      <div class="row">
-          <div class="col-lg-12">
-            {% if ancestors %}
-              <ol class="breadcrumb">
+    <div class="row">
+      <div class="col-lg-12">
+        {% if ancestors %}
+        <ol class="breadcrumb">
 
-                <li><a href="/"><i class="fa fa-home" aria-hidden="true"></i><span class="hidden" aria-hidden="false">Home</span></a></li>
-                  {% for ancestor in ancestors|slice:"1:" %}
-                    {% if forloop.last %}
-                      <li class="active" aria-level="{{ancestors|length}}">
-                        {{ ancestor }}
-                      </li>
-                    {% else %}
-                      <li><a href="{% pageurl ancestor %}" aria-level="{% cycle 1 2 3 4 5 %}">{{ ancestor }}</a></li>  
-                    {% endif %}
-                  {% endfor %}
-              </ol>
-            {% endif %}
-          </div>
+          <li><a href="/">Home</a> {% include "includes/chevron-icon.html" with class="breadcrumb__chevron-icon" %}</li>
+          {% for ancestor in ancestors|slice:"1:" %}
+          {% if forloop.last %}
+          <li class="active" aria-level="{{ancestors|length}}">
+            {{ ancestor }}
+          </li>
+          {% else %}
+          <li><a href="{% pageurl ancestor %}" aria-level="{% cycle 1 2 3 4 5 %}">{{ ancestor }}</a>
```

Comment:
aria-level is for headings, it likely does nothing for links so should be removed.

Reply:
Removed!